sdk/keyvault/azkeys/CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 # Release History
 
-## 0.1.1 (Unreleased)
+## 0.2.0 (Unreleased)
 
 ### Features Added
+* Adds the `CreateOKPKey` method for creating OKP public keys
+* Adds the `Ed25519` `JSONWebKeyCurveName`
+*Adds the `OKP` and `OKPHSM` `JSONWebKeyType`s
```

Comment:
1. Did the "JSONWeb" prefix already ship? None of the languages use that, just `KeyCurveName` and `KeyType`.
2. Missing a space required for a bulleted list on line 8.

Reply:
Yes a first beta was released. The name can be changed to align with the other Tier 1 languages.

Reply:
Aligning with other languages would be ideal if no good idiomatic reason exists to differ.